client/src/index.js

```diff
@@ -1,39 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
It's good practise to remove code that is not used. Since `useState` is not used, let's remove it.